src/+energy/calculateEnergyFluxes.m

```diff
@@ -0,0 +1,10 @@
+function [QET, QEB] = calculateEnergyFluxes(SAVE, TT)
+    %{
+        Calculate the energy fluxes on the boundary nodes.
```

Comment:
Here a documentation reference is missing as well

Reply:
Looking at issue #100, I assume the reference is STEMMUS Technical Notes. Added.